test/powershell/Language/Scripting/Debugging/Debugging.Tests.ps1

```diff
@@ -331,6 +331,38 @@ elseif (Test-Path $PSCommandPath)
             $Breakpoint.HitCount | Should -Be $HitCount
         }
     }
+
+    Context "Break point on return should hit" {
+        BeforeAll {
+            $return_script_1 = @'
+return
+'@
+            $return_script_2 = @'
+return 10
+'@
```

Comment:
Can you please add one more test case to cover `return` statement in a trap block?

Reply:
Done! Also added disassembly for traps in the description.

Reply:
Probably should have waited for the tests to pass before saying that. Looking into the failure

Reply:
Thanks for the additional fix. Will take a look today.